ambari-server/src/main/resources/stacks/BIGTOP/3.3.0/services/RANGER/package/scripts/setup_ranger_xml.py

```diff
@@ -465,24 +465,43 @@ def password_validation(password):
 def copy_jdbc_connector(ranger_home):
   import params
 
-  if params.jdbc_jar_name is None and params.driver_curl_source.endswith("/None"):
+  if params.jdbc_jar_name is None or params.driver_curl_source.endswith("/None"):
```

Comment:
If `jdbc_jar_name` is `None`, `driver_curl_source` ends with `/None` because `driver_curl_source = format("{jdk_location}/{jdbc_jar_name}")` at params.py.